src/main/java/com/example/onlinebookstore/repository/RoleRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.onlinebookstore.repository;
+
+import com.example.onlinebookstore.model.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Optional<Role> findByName(Role.RoleName name);
```

Comment:
Maybe you should use getByName() so that you won't have Optional here

Reply:
Not fixed